Python/context.c

```diff
@@ -1179,10 +1179,17 @@ static PyGetSetDef PyContextTokenType_getsetlist[] = {
     {NULL}
 };
 
+static PyMethodDef PyContextTokenType_methods[] = {
+    {"__class_getitem__",    (PyCFunction)Py_GenericAlias,
+    METH_O|METH_CLASS,       PyDoc_STR("See PEP 585")},
```

Comment:
Overall looks good, but I don't think this is a good docstring. Something like "A class-level __getitem__ to make Token a generic" would make more sense to me.

Reply:
Fair enough, its what was being used for all the other PEP 585 docstrings. I will go ahead and change to your suggested text.